packages/admin/admin/src/mui/menu/GroupSection.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, ComponentsOverrides, Theme, Typography } from "@mui/material";
+import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import * as React from "react";
+
+export type MenuGroupSectionClassKey = "root" | "sectionTitle";
+
+const styles = (theme: Theme) =>
+    createStyles<MenuGroupSectionClassKey, MenuGroupSectionProps>({
+        root: { marginTop: theme.spacing(8) },
+        sectionTitle: {
+            fontWeight: theme.typography.fontWeightBold,
+            fontSize: 14,
+            lineHeight: "20px",
+            borderBottom: `1px solid ${theme.palette.grey[50]}`,
+        },
+    });
+
+export interface MenuGroupSectionProps {
+    title?: string;
```

Comment:
Please use `React.ReactNode` for better translatability. And the title should be enforce IMO

Reply:
I what would I wrap the ReactNode then? Shouldn't be a Typography but then I think it's not a good UX...

Reply:
Please keep Typography, which is perfectly fine, as we want the title to always have the same styling (which may be adapted via the theme).

We're using `React.ReactNode` to support `title={<FormattedMessage id="..." />}` instead of only `title={intl.formatMessage({ id: "..." })}`, which is simpler to use and may be even used in places where `useIntl()` can't be used (e.g., a class component 😱 )

Reply:
Ahh alright got it 👍🏻